### PR TITLE
add zh_cn translation for error

### PR DESCRIPTION
**Pull request by WangMengabc**
translation   tools/crm_resource.c file error message

### CODE REVIEW

File: po/zh_CN.po [hunk 1]
```diff
@@ -814,6 +814,83 @@ msgid ""
...
+
+#: tools/crm_resource.c:1155
+#, c-format
+msgid "No agents found for standard '%s'"
+msgstr "没有发现指定的'%s'标准代理"
+
+#: tools/crm_resource.c:1158
+#, fuzzy, c-format
+msgid "No agents found for standard '%s' and provider '%s'"
+msgstr "没有发现指定的'%s'标准代理"
```
Comment (gao-yan) [inline po/zh_CN.po:865]: Somehow this got imprecise anymore and basically it doesn't match the original format.

**Comment by wenningerk** [comment]
test this please

**Review by gao-yan — CHANGES_REQUESTED** [review]
AFAICS, this basically is https://github.com/ClusterLabs/pacemaker/pull/2941 ? There's a regression here though compared with the other. Please check the inline comment.

BTW, I believe by arranging your local branch with "git reset" and "git commit --amend" commands, and then "git push " again, the other PR could have been updated ... For example like demonstrated here:

https://github.com/ClusterLabs/pacemaker/pull/2936#pullrequestreview-1184360641

But anyways, since you've already closed the other one and opened this, we can try to make it here :-)

For the future, a generally better idea is to create a separate local branch to work on your project and leave the main branch. So that in case rebase is needed, you will still have the clean main branch for fetching from the upstream.

File: po/zh_CN.po [hunk 2]
```diff
@@ -814,6 +814,83 @@ msgid ""
...
+"若要阻止'%s'在特定位置运行，请指定一个节点。若要防止'%s'在指定位置升级，指定"
+"一个节点并使用--promoted选项"
+
+#: tools/crm_resource.c:1003
+#, c-format
+msgid ""
+"Resource '%s' not moved: active in %d locations.\n"
+"To prevent '%s' from running on a specific location, specify a node."
+msgstr ""
+"资源%s未移动，在%d个位置运行\n"
```
Comment (gao-yan) [inline po/zh_CN.po:845]: Better still use "：“ here.

**Review by gao-yan — APPROVED** [review]
LGTM.

**Pull request merged**